servercom/servercom.go

```diff
@@ -0,0 +1,27 @@
+package servercom
```

Comment:
This package does everything that I mentioned, well done! Now I think we should look at how to make it clean/easy to use from the outside. As I mentioned above, I think it might be better to define our own `struct` that has a `*mm.client` as a field. This would allow us to write custom methods that add a level of convenience on top of the basic client that mattermost wrote for us. Thoughts?